src/controllers/userRole/index.js

```diff
@@ -12,17 +12,17 @@ const newUserRoleLink = async (req, res, next) => {
   }
 }
 
-const getUserRoleLinkByUser = async (req, res, next) => {
+const getAllUserRoleLinksByUser = async (req, res, next) => {
   try {
-    const listOfUserRole = await userRoleDomain.getByUserId(req.params.userId)
+    const listOfUserRole = await userRoleDomain.getByUserId(req.params.userId, res.locals.lazyload)
```

Comment:
I think it is not necessary to have lazyload implemented here because one user will not have more than 20 roles for example

Reply:
i thought about this, and i agreed!